src/sql/platform/dialog/dialogPane.ts

```diff
@@ -101,10 +117,14 @@ export class DialogPane extends Disposable implements IThemable {
 					if (tab) {
 						tab.notifyValidityChanged(valid);
 					}
-				}
+				},
+				onLayoutRequested: this._onTabChange.event
 			} as DialogComponentParams,
 			undefined,
-			(moduleRef) => this._moduleRefs.push(moduleRef));
+			(moduleRef) => {
+				this._dialogModule = moduleRef;
```

Comment:
What's the purpose of `this._dialogModule`? I don't see it used anywhere. If it is, it should be updated when the tab changes since there will be a single `moduleRef` for each tab